src/kristin_tukun_ex/ex_02/bubble_sort.py

```diff
@@ -0,0 +1,24 @@
+def bubble_sort(data):
+    """
+    A function that has a tuple as an input and then sorts this list by comparing all elements in for loops.
```

Comment:
This line is too long, > 79 character, which is PEP8 violation (I noticed that twice by email)